helm/values.yaml

```diff
@@ -0,0 +1,216 @@
+#global values to be used across charts
+global:
+  # cloud.enabled creates the nginx-ingress chart
+  cloud:
+    enabled: false
+    # provisioner: kubernetes.io/aws-ebs
+
+  # values used to define the StorageClass template
+  provisioner: k8s.io/minikube-hostpath # kubernetes.io/aws-ebs 
```

Comment:
Change provisioner in GCE in values.yaml . This is not mentioned in the docs . I think this should be automated. If the global value is aws or gke or azure/